### PR TITLE
Add elemental-core

### PR DESCRIPTION
**Pull request by artPlusPlus**

### CODE REVIEW

File: recipes/elemental-core/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{% set name = "elemental-core" %}
+{% set version = "0.2.1.dev0" %}
```
Comment (jakirkham) [inline recipes/elemental-core/meta.yaml]: Is there an official (non-dev) version we can use?

File: recipes/elemental-core/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+{% set md5 = "3244b9a1d62b4e180c37cea88db5931c" %}
```
Comment (jakirkham) [inline recipes/elemental-core/meta.yaml]: One can use `sha256` from PyPI.

xref: https://pypi.org/project/elemental-core/0.2.1.dev0/#files

File: recipes/elemental-core/meta.yaml (hunk 1 above)
Comment (artPlusPlus) [inline recipes/elemental-core/meta.yaml]: Right now, there is not. This is a component of a side project that gets attention whenever I can give it. I was hoping to use conda-forge to reduce my time spent configuring CI solutions and increase development time on the actual product.

However, I completely understand if conda-forge prefers packages that have reached a certain stability. Did I miss this in the documentation somewhere? If so, I sincerely apologize for adding to your already considerable maintenance burden.

What level of stability is conda-forge aiming to publish? I would be happy to hold off submitting packages until they reach that level.

File: recipes/elemental-core/meta.yaml (hunk 2 above)
Comment (artPlusPlus) [inline recipes/elemental-core/meta.yaml]: Thank you for the link, that is very useful. I had heard about warehouse, but did not realize it was it up and running.

Comment (artPlusPlus) [inline recipes/elemental-core/meta.yaml]: The PyPI and sha256 info has been added to the docs.

artPlusPlus/conda-forge.github.io#348

**Comment by artPlusPlus** [comment]
Closing after discussion in conda-forge/conda-forge.github.io#354.

**Pull request closed without merging**